src/components/UserInfo.js

```diff
@@ -13,7 +14,11 @@ export default class UserInfo {
   }
 
   setUserInfo(data) {
-    this._nameElement.textContent = data.title;
+    this._nameElement.textContent = data.name;
     this._descriptionElement.textContent = data.description;
   }
+
+  setUserAvatar(avatar) {
```

Comment:
It's great to have a separate setter for the avatar, since it's controlled by a different form